src/app/clusters/network-commissioning/network-commissioning.h

```diff
@@ -66,16 +66,12 @@ class Instance : public CommandHandlerInterface,
                     DeviceLayer::NetworkCommissioning::ThreadScanResponseIterator * networks) override;
 
 private:
-    static constexpr uint32_t kFeatureMapWiFi     = 0b0000'0001;
-    static constexpr uint32_t kFeatureMapThread   = 0b0000'0010;
-    static constexpr uint32_t kFeatureMapEthernet = 0b0000'0100;
-
-    static constexpr uint32_t kFeatureMapWireless = 0b0000'0011;
-
     static void _OnCommissioningComplete(const DeviceLayer::ChipDeviceEvent * event, intptr_t arg);
     void OnCommissioningComplete(CHIP_ERROR err);
 
-    const uint32_t mType;
+    using NetworkCommissioningFeature = chip::app::Clusters::NetworkCommissioning::NetworkCommissioningFeature;
```

Comment:
Why is this needed?  This is all inside the `chip::app::Clusters::NetworkCommissioning` namespace, so you can just use `NetworkCommissioningFeature` without this `using` bit.

Reply:
Done.